### ISSUE TITLE
[TASK] Update tags in Details page to meet accessibility

### ISSUE THREAD

**Issue by natdink** [SEMOSS/community#110]
The tag that displays in the details panel is showing in green with black text which does not meet accessibility standards.

The text color should be white. Can we update the base component (assuming there is one) so this won't happen again?

![Image](https://github.com/user-attachments/assets/fb5621d5-50c2-4a88-bf39-791cf34bd3e7)

**Comment by johbaxter** [comment]
Will need to account for colors on each of the tags.

Order on each of the colors

**Comment by johbaxter** [comment]
Lets assign Robert Nubla to this.  Need to ensure git access is granted
